<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-7N9D6N5" failures="3" tests="19" name=" all Test" time="69.077" errors="0" timestamp="2022-12-10T20:24:56 IST">
  <testcase classname="tutorial1.demo3" name="pimFunctionality" time="0.018">
    <failure type="java.lang.AssertionError" message="expected [abs] but found [neo]">
      <![CDATA[java.lang.AssertionError: expected [abs] but found [neo]
at tutorial1.demo3.pimFunctionality(demo3.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- pimFunctionality -->
  <testcase classname="tutorial1.demo3" name="closeBrowser" time="0.006"/>
  <testcase classname="tutorial1.demo3" name="logotfunctionality" time="0.0">
    <skipped/>
  </testcase> <!-- logotfunctionality -->
  <testcase classname="tutorial1.demo3" name="loginFunctionality" time="0.001"/>
  <testcase classname="tutorial1.demo3" name="homeFunctionality" time="0.004"/>
  <testcase classname="tutorial1.demo1" name="opm" time="0.002"/>
  <testcase classname="tutorial1.demo1" name="xyz" time="0.005"/>
  <testcase classname="tutorial1.demo1" name="abc" time="0.001"/>
  <testcase classname="tutorial1.demo6" name="homePage" time="14.365"/>
  <testcase classname="tutorial1.demo6" name="marketPlace" time="14.683">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tutorial1.demo6.marketPlace(demo6.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- marketPlace -->
  <testcase classname="tutorial1.demo4" name="setUp" time="3.374"/>
  <testcase classname="tutorial1.demo4" name="loginFunctionality" time="11.492"/>
  <testcase classname="tutorial1.demo4" name="verifyTitle" time="0.038"/>
  <testcase classname="tutorial1.demo4" name="LogoutFunctinality" time="7.734"/>
  <testcase classname="tutorial1.demo4" name="tearDown" time="0.5"/>
  <testcase classname="tutorial1.demo5" name="setUp" time="3.937"/>
  <testcase classname="tutorial1.demo5" name="loginFunctionality" time="0.354">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at tutorial1.demo5.loginFunctionality(demo5.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginFunctionality -->
  <testcase classname="tutorial1.demo5" name="verifyTitle" time="0.0">
    <skipped/>
  </testcase> <!-- verifyTitle -->
  <testcase classname="tutorial1.demo5" name="tearDown" time="0.32"/>
  <testcase classname="tutorial1.demo2" name="LogoutFunctinality" time="0.0">
    <ignored/>
  </testcase> <!-- LogoutFunctinality -->
  <testcase classname="tutorial1.demo1" name="pqr" time="0.0">
    <ignored/>
  </testcase> <!-- pqr -->
  <testcase classname="tutorial1.demo2" name="tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- tearDown -->
  <testcase classname="tutorial1.demo2" name="loginFunctionality" time="0.0">
    <ignored/>
  </testcase> <!-- loginFunctionality -->
  <testcase classname="tutorial1.demo2" name="verifyTitle" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTitle -->
  <testcase classname="tutorial1.demo2" name="setUp" time="0.0">
    <ignored/>
  </testcase> <!-- setUp -->
</testsuite> <!--  all Test -->
